libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -411,6 +404,12 @@ func CheckCpushares(path string, c uint64) error {
 	return nil
 }
 
+func (m *manager) GetPaths() map[string]string {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	return m.paths
```

Comment:
Not sure if the locking here is needed. Might help in the case when another goroutine is populating the paths in `Apply()`, but practically I don't think this is a possibility. Nevertheless, nothing wrong to have it here.